checksum/dpu/src/task.c

```diff
@@ -67,23 +68,29 @@ static unsigned int compute_checksum(uint8_t *buffer) {
 int task_main() {
     unsigned int tasklet_id = me();
     unsigned int file_size = *((uint32_t *) sys_mbox_recv());
+    if (tasklet_id == 1) /* Initialize once the cycle counter */
```

Comment:
The first tasklet should be tasklet#0 (ie thread 1), not tasklet#1?

Reply:
Updated